### PR TITLE
Presets

### PR DESCRIPTION
**Pull request by osse101**
I went through the list and here's how I see things.  Very hard is like a solid competitive rando racer.  Expert has done some hard strat grinding.  Hard is new to competitive racing.

I'll rework how CF clips are structured as techs.

Hotarubi special as a name is very misleading especially if it is moved out of very hard.

### CODE REVIEW

File: rust/data/presets.json [hunk 1]
```diff
@@ -52,46 +54,46 @@
...
-      "canXrayWaitForIFrames",
+      "canXRayWaitForIFrames",
       "canDownBack",
       "canPlayInSand",
       "canHyperGateShot",
       "canSpaceJumpWaterBounce",
       "canWaterShineCharge",
-      "canHeroShot"
+      "canHeroShot",
+      "canMaridiaTubeClip"
```
Comment (kjbranch) [inline rust/data/presets.json:80]: I'm thinking tube clip is obscure enough to be in VH

File: rust/data/presets.json [hunk 2]
```diff
@@ -52,46 +54,46 @@
...
     ],
     "notable_strats": [
       "Early Supers Mockball",
       "Alcatraz Escape",
       "Lower Norfair Pillar Room with Power Bombs",
+      "LN Spring Ball Maze Hotarubi Special",
```
Comment (kjbranch) [inline rust/data/presets.json:86]: Hotorubi only uses 26 tiles. Hard requires 27

File: rust/data/presets.json [hunk 3]
```diff
@@ -106,24 +108,22 @@
...
       "canSpringBallBounce",
       "canSuitlessLavaDive",
       "canHitbox",
       "can3HighMidAirMorph",
       "canTurnaroundAimCancel",
       "canTrickyUseFrozenEnemies",
       "canMetroidAvoid",
-      "canMaridiaTubeClip",
       "canGrappleJump",
+      "canSnailClimb",
```
Comment (kjbranch) [inline rust/data/presets.json:119]: On the fence about this one. It seems easier than most of the tech in `Hard` to me, but people struggle with it.

File: rust/data/presets.json [hunk 4]
```diff
@@ -229,29 +225,31 @@
       "canTrickyDashJump",
       "canCWJ",
       "canHBJ",
-      "canUnmorphBombBoost",
+      "canDoubleBombJump",
```
Comment (kjbranch) [inline rust/data/presets.json:228]: I'm surprised about this one. Using it efficiently or for a large climb could be a bit tricky. One place that this is used is for getting the Blue Brinstar ETank with a jump into IBJ, which I'm not thinking should be Expert.

File: rust/data/presets.json [hunk 5]
```diff
@@ -229,29 +225,31 @@
...
       "canContinuousDboost",
       "can2HighWallMidAirMorph",
       "canPixelPerfectIceClip",
       "canBabyMetroidAvoid",
       "canBombJumpWaterEscape",
+      "canEscapeSand",
```
Comment (kjbranch) [inline rust/data/presets.json:234]: This one is not too hard if you know how. I think that's more fitting for `VH`

File: rust/data/presets.json (hunk 4 above)
Comment (osse101) [reply rust/data/presets.json:228]: Very few people are comfortable with double bomb jumps.

Ceiling Etank is closer to a hard single IBJ techwise imo.  But there is another way we could write it that works.  3 single bombs following a jump and an unmorph.  And that isn't a unmorphBombBoost but instead a hitboxExtend.

File: rust/data/presets.json (hunk 5 above)
Comment (osse101) [reply rust/data/presets.json:234]: Actually, yeah.  I just know that most people can't do it, but that is more about not trying to learn, probably.

File: rust/data/presets.json (hunk 3 above)
Comment (osse101) [reply rust/data/presets.json:119]: Maybe a better way is to make versions of snailclimb strats with HiJump or springball.  And that would move itemless snail climb to notable.  Either way, it's not fun to see aqueduct early in maprando seeds with no items.  
Reaching top left door is the standout problem.  But I could see right door being changed too.

File: rust/data/presets.json (hunk 2 above)
Comment (osse101) [reply rust/data/presets.json:86]: 27 with 2 openends 😏 (fix strat to say 2 openends)
The quickcharge part does feel like what you want for hard though, a small one tap.

I'd rather see this notable removed.  But at the same time it is unique.

File: rust/data/presets.json (hunk 1 above)
Comment (osse101) [reply rust/data/presets.json:80]: Obscure enough to not put in Hard, Trivial enough to not add to VeryHard 🤣 
I can leave it in VH

File: rust/data/presets.json (hunk 2 above)
Comment (kjbranch) [reply rust/data/presets.json:86]: After practicing it a bit, I still think this one should be `VH`. It's about the minimum runway I would expect with `H`, plus a moderately tight jump, plus an air ball, plus pretty obscure.

Although, I do think we should double check the difficulty requirements of different shinespark runways. If this strat is indeed a 26 runway, then I think `H` should be moved from 27 to 26.

**Pull request merged**